### PR TITLE
Incorporating changes from Standard r7479-7579

### PR DESCRIPTION
**Pull request by Ranran-the-JuicyPork**

### CODE REVIEW

File: simtypes.h [hunk 1]
```diff
@@ -32,6 +32,26 @@
...
+#else
+#	define DELETED
+#endif
+
+#if CXX11(4, 7, 1400)
+#	define OVERRIDE override
+#else
+#	define OVERRIDE
+#endif
+
```
[the remark this answers is not in the document]
Comment (Ranran-the-JuicyPork) [reply simtypes.h]: Thank you for your code review. It is most helpful.

**Comment by jamespetts** [comment]
Thank you for this. Can I check what level of testing that this has had and whether it is ready to be integrated yet?

**Comment by Ranran-the-JuicyPork** [comment]
I am working on up to r7718 and gameplay confirmation has already been done.
The new one is pull request #267.
Please make a final confirmation.

**Comment by Ranran-the-JuicyPork** [comment]
The latest version is pull request #268

**Pull request closed without merging**